deps/CMakeLists.txt

```diff
@@ -23,10 +23,7 @@ ${CMAKE_Fortran_DEBUG_FLAGS}"
   )
 elseif ("${CMAKE_Fortran_COMPILER_ID}" MATCHES "Intel")
   set(OPS_DEFAULT_Fortran_FLAGS "\
--warn all \
--warn errors \
--warn notruncated_source \
--warn nointerfaces \
+-warn alignments,errors,declarations,general,ignore_loc,nointerfaces,notruncated_source,shape,stderrors,uncalled,unused,usage \
```

Comment:
Tiny point: you could swap `errors` and `declarations` to put it in alphabetical order.